### PR TITLE
Querying binance trades for delisted market won't kill entire query

### PR DESCRIPTION
**Pull request by LefterisJP**
Querying binance trades for delisted market won't kill entire query

### CODE REVIEW

File: rotkehlchen/exchanges/binance.py [hunk 1]
```diff
@@ -358,9 +358,18 @@ def api_query(
                 except JSONDecodeError:
                     pass
 
+                if 'Invalid symbol' in msg and method == 'myTrades':
+                    assert options, 'We always provide options for myTrades call'
```
Comment (yabirgb) [inline rotkehlchen/exchanges/binance.py:362]: nitpick: We always prefer to be explicit so I guess would be better
```suggestion
                    assert options is not None, 'We always provide options for myTrades call'
```

Comment (LefterisJP) [reply rotkehlchen/exchanges/binance.py:362]: True but I won't rerun all tests just for this.

**Review by yabirgb — APPROVED** [review]
Approving it. THe failing tests are 500 errors calling covalent + the liquity address I used in tests seems to have moved his allocations so I need to update that test

**Comment by LefterisJP** [comment]
Avax tests failed due to covalent being temporarily unavailable: 

```
Request successful response={"result": null, "message": "Covalent API request https://api.covalenthq.com/v1/43114/address/0x0Ee14eF23b6693e61180e047643e04d4fD3173b4/balances_v2/?key=ckey_cb70cd52c833489ea522f104e1a&limit=1000&page-size=8000 returned invalid JSON response: <html>\n<head><title>503 Service Temporarily Unavailable</title></head>\n<body>\n<center><h1>503 Service Temporarily Unavailable</h1></center>\n<hr><center>nginx/1.19.6</center>\n</body>\n</html>\n"}, status_code=502
```

No idea why that single liquity test failed but you said it was also elsewhere @yabirgb and you looking at it? So not related to my code.

**Pull request merged**